<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cmainguiClass</class>
 <widget class="QMainWindow" name="cmainguiClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>544</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1">
       <widget class="QSpinBox" name="spinModelSize"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Size:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QPushButton" name="btnModelArea">
        <property name="text">
         <string>New Region</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QDoubleSpinBox" name="spinModelScale">
        <property name="decimals">
         <number>4</number>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Scale:</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1" rowspan="6">
     <widget class="QMdiArea" name="mdiArea"/>
    </item>
    <item row="1" column="0">
     <widget class="QListView" name="listOpenFiles"/>
    </item>
    <item row="2" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="btnLoadData">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Load Data</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnRemoveData">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Remove Data</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="0">
     <widget class="QTreeView" name="treeModels">
      <attribute name="headerMinimumSectionSize">
       <number>1</number>
      </attribute>
      <attribute name="headerMinimumSectionSize">
       <number>1</number>
      </attribute>
     </widget>
    </item>
    <item row="4" column="0">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QPushButton" name="btnAddModel">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Add Model</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="btnDeleteModel">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Delete Model</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="5" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="elideMode">
       <enum>Qt::ElideNone</enum>
      </property>
      <property name="usesScrollButtons">
       <bool>true</bool>
      </property>
      <property name="documentMode">
       <bool>false</bool>
      </property>
      <property name="tabsClosable">
       <bool>false</bool>
      </property>
      <property name="movable">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="tabWidgetPage1" native="true">
       <attribute name="title">
        <string>Animation</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <property name="margin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Time Step:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QDoubleSpinBox" name="spinTimestep"/>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="btnStartStop">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
            <horstretch>85</horstretch>
            <verstretch>27</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Start</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Total Time</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QDoubleSpinBox" name="spinTotalTime">
          <property name="maximum">
           <double>100000.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>100.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QPushButton" name="btnReset">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Reset</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="btnSavePhotometry">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Save Photometry</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabWidgetPage2">
       <attribute name="title">
        <string>Minimizer</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QComboBox" name="cboMinimizers"/>
        </item>
        <item>
         <widget class="QPushButton" name="btnMinimizer">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Run Minimizer</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFIle">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionSave"/>
    <addaction name="actionOpen"/>
   </widget>
   <addaction name="menuFIle"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
